pages/docs/tutorials/command-line.md

```diff
@@ -24,6 +24,11 @@ Simply run the following in a terminal and follow any instructions:
 ```bash
 $ curl -s https://get.sdkman.io | bash
 ```
+The above command works for ZSH shells as well. Following it up with the `source` linnking should get `sdkman` working in your current terminal shell
```

Comment:
We want to keep this instruction focused on Kotlin, so let's not overload the instruction with details about SDKMAN.
Instead of the lines you add, I suggest mentioning ZSH right after the list of supported OS. Something like this:
"An easier way to install Kotlin on UNIX based systems such as OS X, Linux, Cygwin, FreeBSD and Solaris is by using SDKMAN!. It also works in Bash and ZSH shells."